src/main.d.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Object whose keys are signal names and values are signal objects.
+ */
+export declare const signalsByName: { [signalName: string]: Signal }
+/**
+ * Object whose keys are signal numbers and values are signal objects.
+ */
+export declare const signalsByNumber: { [signalNumber: number]: Signal }
+
+export declare type SignalAction =
+  | 'terminate'
+  | 'core'
+  | 'ignore'
+  | 'pause'
+  | 'unpause'
+export declare type SignalStandard =
+  | 'ansi'
+  | 'posix'
+  | 'bsd'
+  | 'systemv'
+  | 'other'
+
+export declare type Signal = {
+  /**
+   * Standard name of the signal, for example 'SIGINT'.
+   */
```

Comment:
Comments above each property so they are nicely displayed in code editor auto completions.